src/DateTime/tests/test_datetime.py

```diff
@@ -220,7 +220,7 @@ def test_compare_methods(self):
 
     def test_compare_methods_none(self):
         # Compare a date to None
-        dt = DateTime('1997/1/1')
+        dt = DateTime(0)
```

Comment:
Please remove this change. All it does is to change the comparison from a date in 1997 to a date in 1970, for no apparent reason.

Reply:
@dataflake Thanks for your comment.

`DateTime(0)` is the only case where some of these assertion fail without changes in this merge request, thus having assertions for `DateTime(0)` here is important to avoid future regressions.

How about keeping existing assertions for `DateTime('1997/1/1')` and adding assertions for `DateTime(0)` ?

Reply:
Yes, then **add** the new assertion and leave the old one alone.

Reply:
OK, I pushed a fixup commit to test on both `DateTime('1997/1/1')` and `DateTime(0)`.